### PR TITLE
ruby on rails

### PR DESCRIPTION
**Pull request by wasif-shehzad**

### CODE REVIEW

File: node/sequelize/.gitignore [hunk 1]
```diff
@@ -1,4 +1,8 @@
 node_modules/
```
Comment (syedshah12tech) [inline node/sequelize/.gitignore]: Please remove this directory

File: ruby/ror/app/views/layouts/mailer.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (syedshah12tech) [inline ruby/ror/app/views/layouts/mailer.html.erb]: Do we need this file? We are not displaying anything

File: ruby/ror/app/views/layouts/mailer.text.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<%= yield %>
```
Comment (syedshah12tech) [inline ruby/ror/app/views/layouts/mailer.text.erb]: same as above

File: ruby/ror/db/migrate/0_enable_pgcrypto_extension.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class EnablePgcryptoExtension < ActiveRecord::Migration[5.2]
```
Comment (syedshah12tech) [inline ruby/ror/db/migrate/0_enable_pgcrypto_extension.rb]: are migrations required? We removed them from sequelize

File: ruby/ror/README.md [hunk 5]
```diff
@@ -1,22 +1,21 @@
-# Build and run
```
Comment (syedshah12tech) [inline ruby/ror/README.md]: verify we are changing the correct file

File: ruby/ror/Gemfile [hunk 6]
```diff
@@ -0,0 +1,45 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.4.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.2.3'
+# Use postgresql as the database for Active Record
```
Comment (mbautin) [inline ruby/ror/Gemfile]: `postgresql` -> `YugaByte DB (PostgreSQL-compatible)`

**Comment by mbautin** [comment]
I am confused why this pull request has any sequelize changes at all. Certainly it should not be conflicting with the sequelize app or removing it.

File: ruby/ror/app/controllers/orders_controller.rb [hunk 7]
```diff
@@ -0,0 +1,53 @@
+class OrdersController < ApplicationController
```
Comment (mbautin) [inline ruby/ror/app/controllers/orders_controller.rb]: Please fix indentation to improve readability.

**Comment by syedshah12tech** [comment]
remove these files from PR

**Comment by mbautin** [comment]
> remove these files from PR

This PR still deletes a lot of sequelize example files. My understanding is that this PR should not modify those files in any way.
Something like this may be necessary in the PR branch:

git checkout origin/master node
git commit node

**Comment by syedshah12tech** [comment]
> > remove these files from PR
> 
> This PR still deletes a lot of sequelize example files. My understanding is that this PR should not modify those files in any way.
> Something like this may be necessary in the PR branch:
> 
> git checkout origin/master node
> git commit node

It does indeed delete alot of sequelize files whereas it should have no relation with those files. At this point I would rather favor rejecting this PR and doing a fresh PR with only RoR related changes. @wasif-shehzad?

**Pull request closed without merging**